tests/operation_selection.rs

```diff
@@ -38,11 +49,45 @@ fn operation_selection_works() {
         building_id: "12".to_string(),
         mountain_name: Some("canigou".to_string()),
     };
+    let _unrelated_variables = unrelated::Variables {
+        building_id: "12".to_string(),
+        mountain_name: Some("canigou".to_string()),
+    };
+
+    let expected_echo = echo::ResponseData {
+        echo: Some("tiramisù".to_string()),
+    };
+    let expected_heights = heights::ResponseData {
+        mountain_height: Some(224),
+        building_height: Some(12),
+    };
 
-    let expected_echo = r##"ResponseData { echo: Some("tiramisù") }"##;
-    let expected_heights =
-        r##"ResponseData { mountain_height: Some(224), building_height: Some(12) }"##;
+    assert_eq!(expected_echo, echo_response_data);
+    assert_eq!(expected_heights, heights_response_data);
```

Comment:
I should add an assertion: `assert_eq!(expected_heights_unrelated, heights_response_data);` (to check that response shapes are the same.